components/requirement-list.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+
+var RequirementRow = React.createClass({
+    propTypes: {
+        text        : React.PropTypes.string.isRequired,
+        icon        : React.PropTypes.string
+    },
+    render: function () {
+        var icon = this.props.icon ? this.props.icon : 'fa fa-check';
+        return (
+            <li>
+                <div className="icon-wrapper">
+                    <span className={icon}></span>
+                </div>
+                <div className="text">
+                    { this.props.text }
+                </div>
+            </li>
+        );
+    }
+});
+
+
+var RequirementsList = React.createClass({
+    propTypes: {
+        list        : React.PropTypes.array.isRequired,
+        className   : React.PropTypes.string,
+        icon        : React.PropTypes.string
+    },
+    render: function () {
+        var className = this.props.className ? 'list-unstyled requirements-list' + this.props.className : 'list-unstyled requirements-list';
+        var icon = this.props.icon ? this.props.icon : 'fa fa-check';
+        var listItems = this.props.list.map(function(item,position){
+            return <RequirementRow key={position + '-' + item} icon={icon} text={item} />
```

Comment:
This is a bad idea for the key: If we reorder the items in this array, using a `sort()` for instance, then React will now think that every item in the array is new, and that all the old ones are gone, so it will delete all the DOM nodes, and then rebuild them, instead of simply swapping the ordering without deleting/creating DOM nodes. If the text in the items is uniquely identifying, `key={item}` is the right key.